Y/Yosys/build_tarballs.jl

```diff
@@ -46,7 +45,7 @@ fi
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = filter!(p -> Sys.isapple(p) || Sys.islinux(p), supported_platforms())
+platforms = filter!(p -> Sys.isapple(p) || Sys.islinux(p), supported_platforms(;experimental=true))
```

Comment:
I think we need to skip armv6l for the time being, we miss some dependencies
```suggestion
platforms = filter!(p -> arch(p) != "armv6l" && Sys.isapple(p) || Sys.islinux(p), supported_platforms(;experimental=true))
```